browse/templates/abs/extra_services.html

```diff
@@ -161,6 +162,7 @@ <h3><a href="{{ dblp.base_url }}">DBLP</a> - CS Bibliography</h3>
     </script>
 {% endif %}
     <div class="full-text">
+      <a name="other"></a>
```

Comment:
Is this a mistake?

Reply:
No, no a mistake, but thanks for double checking. Thats just an anchor link that the mobile button jumps the user to if there is no PDF.